tools/validator-client-scripts/envs/mainnet.env

```diff
@@ -0,0 +1,20 @@
+CONFIGURED=true
+
+RUST_LOG=info
+RUST_BACKTRACE=1
+
+BECH32_PREFIX=n
+MIX_DENOM=unym
+MIX_DENOM_DISPLAY=nym
+STAKE_DENOM=unyx
+STAKE_DENOM_DISPLAY=nyx
+DENOMS_EXPONENT=6
```

Comment:
This is just nitpicking, but in _theory_ the tokens could have different exponents (but it's such a niche case we can ignore it)

Reply:
nice to make not of it.